src/App.js

```diff
@@ -1,12 +1,28 @@
 import './App.css';
+import {
+  Routes, Route, Link,
+} from 'react-router-dom';
 import Quote from './components/QuoteApi';
 import Calculator from './components/Calculator';
+import HomePage from './components/Home';
 
 function App() {
   return (
+
     <>
-      <Calculator />
-      <Quote />
+      <nav className="navbar">
+        <h2 className="nav-text">Math Magicians</h2>
+        <ul className="list-cover">
+          <li className="list"><Link className="link-tag" to="/">Home</Link></li>
+          <li className="list"><Link className="link-tag" to="/calculator">Calculator</Link></li>
+          <li className="list"><Link className="link-tag" to="/quotes">Quote</Link></li>
+        </ul>
+      </nav>
```

Comment:
- [ ] [OPTIONAL] It would be nice if this is a component of its own and imported here :+1: